scripts/fluentui-publish/jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     '^.+\\.tsx?$': ['@swc/jest', {}],
   },
   coverageDirectory: './coverage',
-  testTimeout: 20000,
+  testTimeout: 60000,
```

Comment:
quite huge but better increase it here then getting intermittent pipeline failures